>>> 'Pkg.add("MatrixNetworks")' log
INFO: Cloning cache of MatrixNetworks from https://github.com/nassarhuda/MatrixNetworks.jl.git
INFO: Installing Docile v0.5.23
INFO: Installing MatrixNetworks v0.0.1
INFO: Package database updated

>>> 'Pkg.test("MatrixNetworks")' log
Julia Version 0.5.0-dev+4330
Commit 493157e (2016-05-26 09:11 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, haswell)
INFO: Testing MatrixNetworks
running /home/vagrant/.julia/v0.5/MatrixNetworks/test/bfs_test.jl ...
running /home/vagrant/.julia/v0.5/MatrixNetworks/test/bipartite_matching_test.jl ...
running /home/vagrant/.julia/v0.5/MatrixNetworks/test/clustercoeffs_test.jl ...
Error During Test
  Test threw an exception of type ErrorException
  Expression: test_function() == true
  type is immutable
   in clustercoeffs(::MatrixNetworks.MatrixNetwork{Int64}, ::Bool, ::Bool) at /home/vagrant/.julia/v0.5/MatrixNetworks/src/clustercoeffs.jl:38
   in clustercoeffs_test() at /home/vagrant/.julia/v0.5/MatrixNetworks/test/clustercoeffs_test.jl:3
   in macro expansion; at /home/vagrant/.julia/v0.5/MatrixNetworks/test/runtests.jl:26 [inlined]
   in anonymous at ./<missing>:?
   in include_from_node1(::String) at ./loading.jl:426
   in process_options(::Base.JLOptions) at ./client.jl:266
   in _start() at ./client.jl:322
ERROR: LoadError: There was an error during testing
 in record(::Base.Test.FallbackTestSet, ::Base.Test.Error) at ./test.jl:321
 in do_test(::Base.Test.Threw, ::Expr) at ./test.jl:219
 in macro expansion; at /home/vagrant/.julia/v0.5/MatrixNetworks/test/runtests.jl:26 [inlined]
 in anonymous at ./<missing>:?
 in include_from_node1(::String) at ./loading.jl:426
 in process_options(::Base.JLOptions) at ./client.jl:266
 in _start() at ./client.jl:322
while loading /home/vagrant/.julia/v0.5/MatrixNetworks/test/runtests.jl, in expression starting on line 21
===========================[ ERROR: MatrixNetworks ]============================

failed process: Process(`/home/vagrant/julia/bin/julia -Cx86-64 -J/home/vagrant/julia/lib/julia/sys.so --compile=yes --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.5/MatrixNetworks/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: MatrixNetworks had test errors
 in #test#49(::Bool, ::Function, ::Array{AbstractString,1}) at ./pkg/entry.jl:693
 in (::Base.Pkg.Entry.#kw##test)(::Array{Any,1}, ::Base.Pkg.Entry.#test, ::Array{AbstractString,1}) at ./null:0
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}})() at ./pkg/dir.jl:31
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}}, ::String) at ./file.jl:59
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./pkg/dir.jl:31
 in (::Base.Pkg.Dir.#kw##cd)(::Array{Any,1}, ::Base.Pkg.Dir.#cd, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./null:0
 in #test#3(::Bool, ::Function, ::String, ::Vararg{String,N}) at ./pkg.jl:245
 in test(::String, ::Vararg{String,N}) at ./pkg.jl:245
 in eval(::Module, ::Any) at ./boot.jl:225
 in process_options(::Base.JLOptions) at ./client.jl:243
 in _start() at ./client.jl:322

>>> End of log
